src/content/self-hosted/install/auth/openid-connect.mdx

```diff
@@ -1,11 +1,13 @@
 ---
-title: Use OpenID Connect as your Authentication Provider
-description: Use OpenID Connect as the authentication provider of your Okteto Self-hosted instance
+title: Use OpenID Connect (OIDC) as your Authentication Provider
+description: Use OpenID Connect (OIDC) as the authentication provider of your Okteto Self-hosted instance
 sidebar_label: OpenID Connect
 id: openid-connect
 ---
 
-This guide will walk you through the steps necessary to configure OpenID Connect as the authentication provider for Okteto.
+This guide will walk you through the steps necessary to configure OpenID Connect (OIDC) as the authentication provider for Okteto.
+
+If you are looking to configure Private Registry credentials from Amazon ECR, please [see our guide here](admin/registry-credentials/amazon-ecr.mdx#using-openid-connect-oidc-federation).
```

Comment:
Could we make this look like a notice or tip?

Reply:
Good call. Updated!